tasks/google/update-fedora-selinux-permissive/task.yaml

```diff
@@ -12,18 +12,21 @@ execute: |
     . "$TESTSLIB/google.sh"
     . "$TESTSLIB/utils.sh"
 
-    # Make the upgrade
-    dnf makecache
-    dnf upgrade -y
-    dnf clean all -y
+    if [ "$SPREAD_REBOOT" = 0 ]; then
+        # Upgrade and clean the system
+        dnf makecache
+        dnf upgrade -y
```

Comment:
Any reason not to include proper entry in `distro_upgrade_packages` for Fedora, Debian and Ubuntu?

Do you need the same for opensuse?

Reply:
No, I'll make that change